pkgs/test/texlive/default.nix

```diff
@@ -613,7 +613,7 @@ rec {
         correctLicenses = scheme: builtins.foldl'
                 (acc: pkg: concatLicenses acc (lib.toList (pkg.meta.license or [])))
                 []
-                scheme.passthru.packages;
+                scheme.passthru.requiredTeXPackages;
```

Comment:
```suggestion
```
TODO: check if `recurseIntoAttrs` on `texlive.pkgs` (better yet `texlivePackages`) makes `find-tarballs.nix` work out of the box, in which case we can delete this line.

Reply:
To be clear: this should not be implemented here. Adding `recurseIntoAttrs` on `texlivePackages` would surface all TeX Live packages in search, thus requiring appropriate `meta.description` and warnings that the packages are for `withPackages`. And maybe there are less obtrusive ways to trick `find-tarballs.nix`.